en-us/api/Microsoft_Toolkit_Uwp_UI_InMemoryStorage.md

```diff
@@ -0,0 +1,64 @@
+---
+permalink: /en-US/api/Microsoft_Toolkit_Uwp_UI_InMemoryStorage.htm
+title: Microsoft.Toolkit.Uwp.UI.InMemoryStorage API 
+description: API page for Microsoft.Toolkit.Uwp.UI.InMemoryStorage
+keywords: windows, app, toolkit, UWP, API
+layout: api
+search.product: eADQiWindows 10XVcnh
+---
+
+
+# InMemoryStorage<T> class
+
+Represents a strongly typed in-memory storage of objects that can be added / removed as required
```

Comment:
Please keep some consistency with the description here, ideally use the same as previous cache docs